ucsmsdk/ucshandle.py

```diff
@@ -235,7 +235,14 @@ def query_dns(self, *dns):
         if not dns:
             raise ValueError("Provide Comma Separated string of Dns")
```

Comment:
@parag-may4 thanks for catching this. have updated the error message and docstring for both methods.